dev/docs/ButtonDocs.md

```diff
@@ -192,20 +214,28 @@ These specialty buttons should be used sparingly and are generally reserved for
 <button class="xy-btn-accent-inverse-lg" disabled>Button Text</button>
 ```
 
-## Deprecated
-
-The following buttons are deprecated and their usage should be replaced.
-
-### White
-
-Most of the existing use cases can be replaced with either `xy-btn-secondary` or `xy-btn-neutral`.
+### Brand Alternative
 
 <DocsDemo>
     <div class="btn-layout">
-        <button class="xy-btn-white">Button Text</button>
+        <button class="xy-btn-alt-sm">Button Text</button>
+        <button class="xy-btn-alt">Button Text</button>
+        <button class="xy-btn-alt-lg">Button Text</button>
+    </div>
+    <div class="btn-layout">
+        <button class="xy-btn-alt-sm" disabled>Button Text</button>
+        <button class="xy-btn-alt" disabled>Button Text</button>
+        <button class="xy-btn-alt-lg" disabled>Button Text</button>
     </div>
 </DocsDemo>
 
 ```html
-<button class="xy-btn-white">Button Text</button>
+<button class="xy-btn-alt-sm">Button Text</button>
+<button class="xy-btn-alt">Button Text</button>
+<button class="xy-btn-alt-lg">Button Text</button>
+
+<!-- disabled -->
+<button class="xy-btn-alt-sm" disabled>Button Text</button>
+<button class="xy-btn-alt" disabled>Button Text</button>
+<button class="xy-btn-alt-lg" disabled>Button Text</button>
```

Comment:
We're close to 100% parity on button styles with marketing at this point, so I don't expect much more to show up here until an emerald/sapphire rebrand bomb drops.